src/components/Sections/SimpleMode/Container.js

```diff
@@ -34,17 +32,10 @@ const SimpleModeContainer = () => {
     return newDate;
   });
 
-  const dispatch = useDispatch();
-
   useEffect(() => {
     setFilesLanguages(contentLanguages);
   }, [contentLanguages]);
 
-  useEffect(() => {
```

Comment:
Why we don't need this any more?

Reply:
RTK handles cache, so as far as parameters are the same - the result is the same. You don't need to react on parameter change - RTK will do it by itself